modules/govuk_chromedriver/manifests/init.pp

```diff
@@ -0,0 +1,18 @@
+# == Class: govuk_chromedriver
+#
+# Installs Google Chrome and its associated ChromeDriver
+# Used by Selenium tests which need to drive a browser
+#
+class govuk_chromedriver {
+  require google_chrome
+
+  # Copy the install-chromedriver.sh script on to the machine
+  file { '/usr/sbin/install-chromedriver':
+    ensure => file,
+    source => 'puppet:///modules/govuk_chromedriver/install-chromedriver.sh',
+    mode   => '0755',
+  } # and then execute it (-> denotes a dependency)
+  -> exec { 'install-chromedriver':
+    command => '/usr/sbin/install-chromedriver',
+  }
+}
```

Comment:
I feel like we should add a rudimentary test to go with this, something like this might work?

```
require_relative '../../../../spec_helper'

describe 'govuk_chromedriver', :type => :class do

  it { is_expected.to compile }

  it { is_expected.to compile.with_all_deps }

  it { is_expected.to contain_class('govuk_chromedrive') }

  it { is_expected.to contain_class(/usr/sbin/install-chromedriver').with_ensure('file') }

  it { is_expected.to contain_exec('install-chromedriver') }
end

```

Reply:
Thanks for the nudge – I'd forgotten about this 🤦🏻

Your test worked pretty much out-the-box, so thank you for this.